### ISSUE TITLE
30mm bolts most likely too short and not rigid

### ISSUE THREAD

**Issue by Akustav** [ut-robotics/picr21-team-bts#99]
IT seems to me that these bolts are too short and will bend inside soft plastic:

![image](https://user-images.githubusercontent.com/5435954/151010321-73e30516-4c0b-4997-b6bb-42827c7f1414.png)

I suggest to use spacers.

**Comment by mbz4** [comment]
True but I dont think a 40mm M3 bolt even exists ._.

**Comment by Akustav** [comment]
Yeah, but there exists 40mm spacers that screw into the holes and don't apply all the force onto the hole. Spacers have a bottom that can rest on top of this component and the friction there transfers forces to the body of the component more evenly

**Comment by Akustav** [comment]
Additionally, what fixes the nuts into the plastic? Threads don't bite into the bolt, that is why nuts are also torqued down. Currently the bolts just would not hold the middle ramp in a stable fashion.
